package/endpoint_adapter/src/endpoint_adapter_can.c

```diff
@@ -192,8 +208,15 @@ static void *can_write_thread_handler(void *arg)
   return NULL;
 }
 
-int can_loop(const char *can_name, u32 can_filter_in)
+int can_loop(const char *can_name,
+             u32 can_filter_in,
+             const char *framer_in_name,
+             const char *framer_out_name)
 {
+  if (strncmp(framer_in_name, "j1939", 5) == 0) {
```

Comment:
I feel like this could be cleaner if there was a j1939 switch for endpoint adapter that could drive an `is_j1939` bool flag that gets passed to this function.  This would require the ports daemon to know about this flag though.

Reply:
I agree it'd be cleaner, but given that we're trying to close out J1939 dev work, I'll add a TODO comment and revisit if/when we resume in the future

Reply:
😭